src/reporting/clients/IMappingsClient.ts

```diff
@@ -0,0 +1,502 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { AccessToken } from "@itwin/core-bentley";
+import { CalculatedProperty, CalculatedPropertyCreate, CalculatedPropertyUpdate } from "../interfaces/CalculatedProperties";
+import { CustomCalculation, CustomCalculationCreate, CustomCalculationUpdate } from "../interfaces/CustomCalculations";
+import { GroupProperty, GroupPropertyCreate, GroupPropertyUpdate } from "../interfaces/GroupProperties";
+import { Group, GroupCreate, GroupUpdate } from "../interfaces/Groups";
+import { Mapping, MappingCreate, MappingUpdate, MappingCopy } from "../interfaces/Mappings";
+import { EntityListIterator } from "../iterators/EntityListIterator";
+
+export interface IMappingsClient {
+  /**
+   * Gets all Mappings for an iModel.
```

Comment:
Add sentence: `This method returns the full list of mappings.` Similar applies to all methods on all clients that return all entities at once with a single call.

Reply:
added